tests/security/create_swtpm_hdd/build_hdd.pm

```diff
@@ -49,6 +50,22 @@ sub run {
     assert_script_run("wget --quiet " . data_url("swtpm/70-persistent-net.rules") . " -O $udev_rule_file");
     assert_script_run("sed -i 's/NAME=\"\"/ NAME=\"$nic_name\"/' $udev_rule_file");
 
+    # Permit ssh login as root
+    # Due to bsc#1173067, openssh now no longer allows RootLogin
+    # using password auth on Tumbleweed, the latest Leap 16 will
+    # sync with Tumbleweed as well
+    if (is_sle or is_leap('<16.0')) {
+        my $results = script_run("grep 'PermitRootLogin yes' /etc/ssh/sshd_config");
+        if (!$results) {
+            assert_script_run("sed -i 's/^PermitRootLogin.*\$/PermitRootLogin yes/' /etc/ssh/sshd_config");
+            assert_script_run("systemctl restart sshd");
+        }
+    }
+    else {
+        assert_script_run("echo 'PermitRootLogin yes' > /etc/ssh/sshd_config.d/root.conf");
+        assert_script_run("systemctl restart sshd");
+    }
+
```

Comment:
Suggest wrap as a function in lib/utils? I suppose maybe this needed for other test cases

Reply:
Code duplicated.

Reply:
Thanks all! it makes sense, let me do that

Reply:
done, please check again